src/utils/fxa.js

```diff
@@ -67,10 +69,11 @@ async function revokeOAuthTokens (subscriber) {
 
 async function getProfileData (accessToken) {
```

Comment:
@mansaj as discussed, this request gets called twice – unrelated to Fetch replacement.  We should look into fixing in case we can avoid a second redundant network request.

Reply:
created a bug ticket: MNTOR-1064